### DISCUSSION TITLE
Audio quality of screen recording

### DISCUSSION THREAD

**Discussion by B1ueH0r1z0n** [obsproject/obs-studio#8248] [Development Questions] [answered]
I hope that OBS studio can improve audio-recording quality. OBS studio can only record sound in aac format for now, and only supports sound quality up to 320kbps. When I observed the spectrum of the recorded sound, I found that OBS would force a low-pass of 18khz when recording, which would inevitably affect the sound quality. I hope that obs can add new options for audio format, for example directly recording pcm and outputting .wav files, or at least not performing any post-processing of audio at 320kbps, such as the 18khz low-pass I just mentioned, thus to improve the audio quality of screen recording.
![屏幕截图(6)](https://user-images.githubusercontent.com/125110233/218243750-d60bf5a0-b6f2-4939-8824-c89fc5afd1b8.jpg)

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds more likely to be a feature request, and we are not currently accepting feature requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion.

Thank you!
